### PR TITLE
Add support thecodingmachine/graphqlite 5

### PR DESCRIPTION
**Pull request by DenisVinokurov**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,9 +16,12 @@
             "homepage": "http://mouf-php.com"
         }
     ],
+    "scripts": {
+        "test": "phpunit"
+    },
     "require": {
-        "php": ">=7.2",
-        "thecodingmachine/graphqlite": "^4",
+        "php": ">=7.2|8.0",
```
Comment (devmaslov) [inline composer.json]: No need to specify 8.0 version, since ">=7.2" covers it.

```suggestion
        "php": ">=7.2",
```

File: composer.json [hunk 2]
```diff
@@ -16,9 +16,12 @@
...
+        "thecodingmachine/graphqlite": "^5",
```
Comment (devmaslov) [inline composer.json]:
```suggestion
        "thecodingmachine/graphqlite": "^5.0",
```

**Review by devmaslov — CHANGES_REQUESTED** [review]
@DenisVinokurov Thanks for the PR. Looks good to me, just a few tweaks are needed before merging.

**Comment by DenisVinokurov** [comment]
@devmaslov added fixes

**Review by devmaslov — APPROVED** [review]

**Pull request merged**